packages/flutter/lib/src/rendering/selection.dart

```diff
@@ -619,6 +768,82 @@ enum SelectionStatus {
   none,
 }
 
+/// The details of a selection.
+///
+/// This object is created by users of the [SelectionListenerSelectionChangedCallback] callback.
```

Comment:
By user do you mean the caller of the callback or creator of the callback?

Reply:
Good point, I meant the caller of the callback here.